docs/src/main/paradox/google-cloud-bigquery-storage.md

```diff
@@ -4,14 +4,14 @@ The BigQuery Storage API offers fast access to BigQuery-managed storage using an
 It is seen as an improvement over the REST API, and bulk data `extract` jobs for accessing BigQuery-managed table data, but doesn't offer any functionality around managing BigQuery resources.
 Further information at the official [Google Cloud documentation website](https://cloud.google.com/bigquery/docs/reference/storage).
 
-This connector communicates to the BigQuery Storage API via the gRPC protocol. The integration between Akka Stream and gRPC is handled by the
-@extref:[Akka gRPC library](akka-grpc:/). Currently, this connector only supports returning each row as an Avro GenericRecord.
+This connector communicates to the BigQuery Storage API via the gRPC protocol. The integration between Apache Pekko Stream and gRPC is handled by the
+@extref:[Apache Pekko gRPC library](akka-grpc:/). Currently, this connector only supports returning each row as an Avro GenericRecord.
```

Comment:
the link here, is it still `akka-grpc:/` ?

Reply:
we don't yet publish pekko-grpc docs so I left these links alone. We have the main pekko lib docs published so that's why I updated their links.

when we start publisng pekko-grpc docs, we can update the links here too.